web-next/app/codex/agent/page.module.css

```
@@ -0,0 +1,814 @@
+  --purple: #8b5cf6;
+  --amber: #f59e0b;
+  --red: #ef4444;
+  --teal: #14b8a6;
+  --text: #f0f0f0;
+  --text2: #b8b8b8;
+  --text3: #959595;
+  --shadow: 0 4px 24px rgba(0, 0, 0, 0.6);
+
+  background: var(--bg);
+  color: var(--text);
+  font-family: "Inter", -apple-system, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)